draft-murillo-whep.md

```diff
@@ -229,15 +229,13 @@ a=msid:- d46fb922-d52a-4e9c-aa87-444eadc1521b
 ~~~~~
 {: title="HTTP POST and PATCH doing SDP O/A example"}
 
-## SDP offer generated by the WHEP endpint
+## SDP offer generated by the WHEP endpoint
 
-If the WHEP player preffers the WHEP Endpoint to generate the SDP offer, the WHEP player will send a POST request without HTTP BODY and an Accept HTTP header of "application/sdp" to the configured WHEP endpoint URL.
+If the WHEP player prefers the WHEP Endpoint to generate the SDP offer, the WHEP player will send a POST request without HTTP BODY and an Accept HTTP header of "application/sdp" to the configured WHEP endpoint URL. The HTTP POST request MUST have a content type of "application/sdp". 
```

Comment:
If the WHEP player is acting as a offerer, the HTTP POST will be empty, and there would be no content type header. That is why the Accept http header is set instead

Reply:
Okay, actually it's from the next line. I'll delete it.